canonical-web-and-design/ubuntu.com
# Update ROS contact-us modal

## Done

- Update ROS contact-us modal

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1vDbdFBrG6tWDUb5z5ltny_7d-GZU78wS8K7Nru2BSj8/edit#)
